code/src/index.css

```diff
@@ -5,9 +5,24 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: teal;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+#root {
+  max-width: 800px;
+  margin: auto;
+  cursor: pointer;
```

Comment:
Since the whole root div has "cursor: pointer", the whole div shows a pointer even over areas that are not clickable, it was a bit confusing - maybe only have pointer on clickable elements?